flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowChainLogService.java

```diff
@@ -55,7 +55,7 @@ public List<FlowChainLog> findByFlowChainIdOrderByCreatedDesc(String flowChainId
     public List<FlowChainLog> getRelatedFlowChainLogs(List<FlowChainLog> sourceFlowChains) {
         Optional<FlowChainLog> flowChainWithParent = sourceFlowChains.stream()
                 .filter(flowChainLog -> StringUtils.isNotBlank(flowChainLog.getParentFlowChainId())).findFirst();
-        FlowChainLog lastFlowChain = sourceFlowChains.stream().max(Comparator.comparing(FlowChainLog::getCreated)).get();
+        FlowChainLog lastFlowChain = sourceFlowChains.stream().max(Comparator.comparing(FlowChainLog::getId)).get();
```

Comment:
I suggest to create a real comparator with first comparing the create timestamp and then Id

Reply:
@grace303303 In your original change you used both timestamp and ID, right. Why is it not used here?

Reply:
@kkalvagadda1 I made the changes to use both. Plz let me know if it looks good. Thanks!